### PR TITLE
Do not resolve exportedNames

### LINKED ISSUE #722: export should not `resolve`
e.g. `export var x` need to expand and codegen to `var x_1; export { x_1 as x }`

forms to check:

- `export var x`
- `export { x }`
- `export { x } from 'm'`

### PR DESCRIPTION
**Pull request by disnet**
Fixes #722.

Compiling `export var x` now compiles to `var x_1; export { x_1 as x }`.

### CODE REVIEW

File: src/sweet-module.js [hunk 1]
```diff
@@ -101,17 +111,59 @@ export default class SweetModule {
...
-        imports.push((item: any));
-      } else if (S.isExportDeclaration(item)) {
-        exports.push((item: any));
-        body.push(item);
-        this.exportedNames = this.exportedNames.concat(extractNames(item));
-        if (S.isExportDefault(item)) {
-          this.defaultExport = Syntax.fromIdentifier('_default');
-          this.exportedNames = this.exportedNames.push(
-            ExpSpec(this.defaultExport)
+      if (item instanceof T.ImportDeclaration) {
```
Comment (gabejohnson) [inline src/sweet-module.js]: Why aren't you using the spec helper predicates here?

File: test/parser/__snapshots__/test-ast.js.snap [hunk 2]
```diff
@@ -69,7 +82,30 @@ Module {
...
+  "loc": null,
+  "type": "Module",
+}
+`;
+
+exports[`includes export with renaming in AST 1`] = `
+Module {
+  "directives": Array [],
+  "items": Array [
+    ExportFrom {
```
Comment (gabejohnson) [inline test/parser/__snapshots__/test-ast.js.snap]: Why are these `ExportFrom`?

File: src/sweet-module.js [hunk 3]
```diff
@@ -101,17 +111,59 @@ export default class SweetModule {
...
+        imports.push(item);
+      } else if (item instanceof T.ExportDeclaration) {
+        if (S.isExport(item)) {
+          let decl = extractDeclaration(item);
+          let stmt = wrapStatement(decl);
+          let names = extractDeclarationNames(decl);
+          body.push(stmt);
+          let exp = new T.ExportFrom({
```
Comment (gabejohnson) [inline src/sweet-module.js]: Just answered my own question from the snapshot. We should add `ExportLocals` to sweet-spec.

File: src/sweet-module.js (hunk 1 above)
Comment (disnet) [inline src/sweet-module.js]: because flow understands `instanceof`

File: src/sweet-module.js (hunk 3 above)
Comment (disnet) [inline src/sweet-module.js]: That was added in shift spec es2016, we're on es6.

Definitely need to update but not in this PR.

**Pull request merged**